docs/sorting-algorithms/bubble-sort.md

```diff
@@ -3,4 +3,153 @@ title: Bubble Sort
 slug: /sorting-algorithms/bubble-sort
 ---
 
-This page would serve our needs for documentation of Bubble Sort.
\ No newline at end of file
+## Introduction
+
+The bubble sort is a simple sorting algorithm which compares two adjacent elements and swaps them if they are not in the correct order.
+
+## Explanation
+
+Let's have an array arr[] where we have the following elements: 5 4 3 2 1
+
+![Imgur](https://i.imgur.com/mrC2YZn.png)
+
+![Imgur](https://i.imgur.com/snYKCMG.png)
+
+![Imgur](https://i.imgur.com/blbuUB2.png)
+
+![Imgur](https://i.imgur.com/XLxdlis.png)
```

Comment:
Did you make these diagrams?

Reply:
Yes I made these diagrams on PowerPoint and then uploaded them to Imgur